lua/entities/gmod_tardis/modules/sh_vortex.lua

```diff
@@ -6,6 +6,10 @@ function ENT:IsVortexEnabled(pilot)
 	return ((not pilot and SERVER) or TARDIS:GetSetting("vortex-enabled",false,pilot)) and IsValid(self:GetPart("vortex")) and (SERVER or self:GetData("vortexmodelvalid"))
 end
 
+ENT:AddHook("VortexEnabled", "demat-fast", function(self, pilot)
+	return !self:GetData("demat-fast",false)
+end)
```

Comment:
```suggestion
ENT:AddHook("VortexEnabled", "demat-fast", function(self, pilot)
	if self:GetData("demat-fast", false) then
		return false
	end
end)
```

Hooks should only return if overriding behaviour

Reply:
Also this hook should maybe be part of the teleport module?